.travis/before_install.sh

```diff
@@ -45,6 +45,11 @@ cd /tmp || die
     sudo -H pip install pexpect || die
     pip install pexpect || die
 
+    # posix-ncp tests
+    pip install scapy || die
```

Comment:
Agree.  The NCP tests trigger on `if OPENTHREAD_ENABLE_NCP` in `tests/scripts/ncp/Makefile.am`.  It looks like posix-distcheck and posix-32-bit configure with --enable-ncp.